modules/migration-mtc-release-notes-1-7-18.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes-1-7.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-18_{context}"]
+= {mtc-full} 1.7.18 release notes
+
+{mtc-first} 1.7.18 is a Container Grade Only (CGO) release, released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17.
```

Comment:
This is a run-on sentence. 

Rewrite as: "{mtc-first} 1.7.18 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17."